lib/src/executable.dart

```diff
@@ -81,8 +71,8 @@ Future<int> runExecutable(Entrypoint entrypoint, String package,
   // "bin".
   if (p.split(executable).length == 1) executable = p.join("bin", executable);
 
-  var executableUrl = await _executableUrl(entrypoint, package, executable,
-      isGlobal: isGlobal, mode: mode);
+  var executableUrl =
```

Comment:
Since we aren't going to be running executables over HTTP anymore, I'd make this a path rather than a URL.

This should also let you clean up some of the logic below that's conditional based on whether or not we're running an executable from the filesystem, since we always are now.

Reply:
Done